airflow/config_templates/default_airflow.cfg

```diff
@@ -349,6 +349,7 @@ smtp_ssl = False
 smtp_port = 25
 smtp_mail_from = airflow@example.com
 
+smtp_mime_from =
```

Comment:
Can you add a short comment saying how/where this is different from the above setting please?